storey/redis_driver.py

```diff
@@ -104,13 +107,9 @@ def redis(self):
                 self._redis = redis.cluster.RedisCluster.from_url(self._redis_url, decode_response=True)
         return self._redis
 
-    async def close(self):
-        if self._redis:
-            self._redis.close()
-
     @staticmethod
     def make_key(key_prefix, *parts):
-        return f"{key_prefix}{''.join([str(p) for p in parts])}"
+        return f"{{{key_prefix}{''.join([str(p) for p in parts])}}}"
```

Comment:
What happened here? This change doesn't look right.

Reply:
It is intentionally - this way you keep all related keys on the same cluster node: https://redis.com/blog/redis-clustering-best-practices-with-keys/